ttnn/cpp/ttnn/operations/conv2d/conv2d.cpp

```diff
@@ -885,6 +895,20 @@ template std::tuple<ttnn::Tensor, uint32_t, uint32_t, ttnn::Tensor, std::optiona
     std::optional<const ttnn::Tensor> bias_tensor,
     std::optional<const Conv2dConfig> conv_config_);
 
+
+    bool Conv2dConfig::is_height_sharded() const
+    {
```

Comment:
opening curly brances don't go to new line (comment applies for whole PR)
You can use clang-format when in doubt to format code for you (Format document shift + opt + F, or Format selection cmd + K + F)

Reply:
I tried clang-format in a different [commit](https://github.com/tenstorrent/tt-metal/pull/11122/commits/d42f94ef1dd712049eb1faa80807bdd6b8f6bd85).